javafx/javafx2.project/src/org/netbeans/modules/javafx2/project/JFXProjectUtils.java

```diff
@@ -1776,5 +1779,17 @@ public Void run() throws Exception {
             }
         }
     }
+
+    private static final String MODULE_INFO = "module-info.java"; // NOI18N
+
+    // as in org.netbeans.modules.maven.api.ModuleInfoUtils.hasModuleInfoInSource;
+    public static boolean hasModuleInfo(SourceGroupSupport support) {
+        for (File sourceRoot : support.getSourceGroupsAsFiles()) {
+            if (new File(sourceRoot, MODULE_INFO).exists()) {
```

Comment:
I would prefer working with `FileObjects`, rather than converting to `File`s, although it does not make probably big difference here. Consider 
```
for (FileObject sourceRoot : support.getSourceGroups()) {
  if (sourceRoot.getFileObject(MODULE_INFO) != null) {
     ...
  }
}

Reply:
Excellent. Thanks. Had to go through SourceGroupProxy, but...

@sdedic could you re-review with suggested changes?